.gitignore

```diff
@@ -338,6 +338,7 @@ build
 *.log
 .dist/
 ./BridgeCareApp/VuejsApp/dist
+package-lock.json
```

Comment:
Need to remove package-lock.json from source for this to work

Reply:
You mean from the GitHub?

Reply:
Deleted the file from Feature/49 branch